internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
+
```

Comment:
Ensure that the routes defined have appropriate error handling. Lack of error handling can lead to unhandled cases when requests fail, which may compromise the user experience.